### PR TITLE
Fixes #8927: Prepare manual for 4.0

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8927

### CODE REVIEW

File: 12_upgrade/00_upgrade.txt [hunk 1]
```diff
@@ -10,16 +10,16 @@ automatically by packaging scripts. Therefore, you shouldn't have to do any
...
-==== Upgrade from Rudder 2.11, 3.0 or 3.1
+==== Upgrade from Rudder 3.1 or 3.2
 
-Migration from 2.11, 3.0 or 3.1 are supported, so you can upgrade directly to 3.2.
+Migration from 3.1 or 3.2 are supported, so you can upgrade directly to 4.0.
 
-If you migrate from 2.11, 3.0 or 3.1 older than 3.1.11, you have to upgrade
-you server before your agents (as 3.2 agents are not compatible with older servers).
+If you migrate from a 3.1 release older than 3.1.11, you have to upgrade
+you server before your agents (as 4.0 agents are not compatible with old servers).
```
Comment (jooooooon) [inline 12_upgrade/00_upgrade.txt]: Are you sure this is not the case in the first 3.2.* releases?

File: 12_upgrade/05_caution.txt [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
+Rudder agents 4.0.x are fully compatible with Rudder server 3.1.x starting from 3.1.11 and 3.2 servers.
 
-===== 2.11.x, 3.0.x and 3.1.x agents
+===== <3.0.x and 3.1.x (< 3.1.11) servers
 
-Rudder agents 3.2.x are not compatible with Rudder server 2.11.x, 3.0.x or
-3.1.x lower than 3.1.11.
+Rudder agents 4.0.x are not compatible with Rudder servers older than than 3.1.11.
 
 ==== Compatibility between Rudder server 3.2 and older agent versions
```
Comment (jooooooon) [inline 12_upgrade/05_caution.txt]: I think this should be 4.0

**Comment by jooooooon** [comment]
Great idea. We'll also need to change the release-info data from 3.3 to 4.0, the versions.sh file in packaging and the CI tasks to publish on these URLs. Could you also do that please?

**Comment by amousset** [comment]
Commit modified

File: 12_upgrade/00_upgrade.txt (hunk 1 above)
Comment (amousset) [inline 12_upgrade/00_upgrade.txt]: Yes, because https://github.com/Normation/rudder-techniques/pull/830 was already done in 3.2.0.

**Comment by amousset** [comment]
Search will not work until the 4.0 doc url is published and we ask for its indexation.

**Comment by jooooooon** [comment]
OK, merging. Don't forget to ask for the 4.0 doc url to be indexed then :)

**Pull request merged**